src/Editor/Image/Panel.tsx

```diff
@@ -1,37 +1,43 @@
-import React, { FC, useState, useMemo } from 'react';
+import React, { FC, useState } from 'react';
 import { CropperRef, Cropper } from 'react-advanced-cropper';
 import { EditorArgs } from '../EditorArgs';
 import { useWhppt } from '../../Context';
-import { WhpptGalleryTag, WhpptInput } from '../../ui/components';
+import { WhpptInput } from '../../ui/components';
+import { WhpptGalleryTag } from '../../Gallery/GalleryTag';
 import { Gallery } from '../../Gallery';
-import { aspectRatios } from './AspectRatios';
-import { getLandscapeRatio, getPortraitRatio } from './helpers';
+import { aspectRatios } from '../../Gallery/Model';
 import { DevicePicker } from './DevicePicker';
-import { ImageDataSize, PageImageData, ImageData } from './../../Gallery/Model/Image';
+import { ImageItemDataSize, PageImageItemData, ImageItemData } from './../../Gallery/Model/Image';
 import { ImageEditorOptions } from '../Editors';
 
-type Orientation = 'landscape' | 'portrait';
+const getLandscapeRatio = ratio => {
+  const { w, h } = ratio;
+  return w >= h ? w / h : h / w;
+};
+
+const getPortraitRatio = ratio => {
+  const { w, h } = ratio;
+  return w >= h ? h / w : w / h;
+};
 
-export const WhpptImageEditorPanel: FC<EditorArgs<PageImageData, ImageEditorOptions>> = ({ value, onChange, options }) => {
+export const WhpptImageEditorPanel: FC<EditorArgs<PageImageItemData, ImageEditorOptions>> = ({ value, onChange, options }) => {
   const { toggleSettingsPanel } = useWhppt();
 
   const [device, setDevice] = useState<string>('desktop');
-  const [orientation, setOrientation] = useState<Orientation>('landscape');
+  const [stencilProps, setStencilProps] = useState(() => {
+    if (!value[device]) return getLandscapeRatio(aspectRatios[0].ratio);
 
-  const selectedDevice = useMemo(() => value && value[device], [device, value]);
```

Comment:
Is there a problem using the memo? It works just wondering why we would use value[] rather than selectedDevice?